### PR TITLE
gcc@9.3.0-1: Fix URL

### LINKED ISSUE #1471: gcc@9.3.0-1: Download failed
Install of gcc on main bucket not working.
```powershell
scoop install gcc
Installing 'gcc' (9.3.0-1) [64bit]
Unable to connect to the remote server
URL http://repo.msys2.org/mingw/x86_64/mingw-w64-x86_64-binutils-2.34-1-any.pkg.tar.xz is not valid

scoop --version
Current Scoop version:
6df80c1a (HEAD -> master, origin/master, origin/HEAD) fix(tests): Force pester v4 (#4040)
'main' bucket:
1cc739df (HEAD -> master, origin/master, origin/HEAD) imagemagick: Update to version 7.0.10-33
```

### PR DESCRIPTION
**Pull request by wstein**
SF does not have all files available (at least the manifest file is missing).
So I choose mirrors.ustc.edu.cn, which seems to be a good performing mirror.

Closes #1471

### CODE REVIEW

**Comment by janymal** [comment]
I think that changing the mirror to SourceForge would perform better as it's not subordinated to one specific location. SourceForge basically has its own mirror network for even serving files across the globe so it would be more universal solution than fixing the mirror to China.

File: bucket/gcc.json [hunk 1]
```diff
@@ -6,42 +6,42 @@
...
+                "http://mirrors.ustc.edu.cn/msys2/mingw/x86_64/mingw-w64-x86_64-make-4.3-1-any.pkg.tar.xz",
+                "http://mirrors.ustc.edu.cn/msys2/mingw/x86_64/mingw-w64-x86_64-mpc-1.1.0-1-any.pkg.tar.xz",
+                "http://mirrors.ustc.edu.cn/msys2/mingw/x86_64/mingw-w64-x86_64-mpfr-4.0.2-2-any.pkg.tar.xz",
+                "http://mirrors.ustc.edu.cn/msys2/mingw/x86_64/mingw-w64-x86_64-pkg-config-0.29.2-1-any.pkg.tar.xz",
+                "http://mirrors.ustc.edu.cn/msys2/mingw/x86_64/mingw-w64-x86_64-windows-default-manifest-6.4-3-any.pkg.tar.xz",
+                "http://mirrors.ustc.edu.cn/msys2/mingw/x86_64/mingw-w64-x86_64-winpthreads-git-8.0.0.5680.0df6b89f-1-any.pkg.tar.xz",
+                "http://mirrors.ustc.edu.cn/msys2/mingw/x86_64/mingw-w64-x86_64-zlib-1.2.11-7-any.pkg.tar.xz",
+                "http://mirrors.ustc.edu.cn/msys2/mingw/x86_64/mingw-w64-x86_64-libsystre-1.0.1-4-any.pkg.tar.xz",
+                "http://mirrors.ustc.edu.cn/msys2/mingw/x86_64/mingw-w64-x86_64-libtre-git-r128.6fb7206-2-any.pkg.tar.xz",
+                "http://mirrors.ustc.edu.cn/msys2/mingw/x86_64/mingw-w64-x86_64-libtasn1-4.9-1-any.pkg.tar.xz",
```
Comment (thfr) [inline bucket/gcc.json:31]: This file is not available any more, it is now `mingw-w64-x86_64-libtasn1-4.15.0-1-any.pkg.tar.xz`

**Review by thfr — CHANGES_REQUESTED** [review]
Seems like there is a gcc-9.3.0-2 available now.

**Comment by wstein** [comment]
http://repo.msys2.org/mingw/x86_64/ is up again, so there is no need to fix the url.

**Pull request closed without merging**